proposals/lip-0049.md

```diff
@@ -273,6 +273,9 @@ def apply(trs: Transaction, ccm: CCM) -> None:
     else:
         crossChainCommand = cross-chain command associated with (ccm.module, ccm.crossChainCommand)
         try:  
+            #Decode and validate the parameterSchema associated with ccm module or crossChainCommand
+            ccmParams = decode(paramsSchema, ccm.params)
```

Comment:
`paramsSchema` is undefined. I would define it in words, similar to what we do for `crossChainCommand`